CONTRIBUTING.md

```diff
@@ -179,16 +179,16 @@ Please follow all style guides to the best of your abilities.
 
 ### Git commit messages
 
-For all commit messages, please use a short phrase that describes the specific change.
-For example, “Add feature to check normalization method string” is much preferred to “change code”.
-When appropriate, reference issues (via `#` plus number) .
+Pycytominer uses [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard for commit messages to aid in automatic changelog generation.
+The easiest way to prepare commit messages that follow this standard is to use [commitizen](https://commitizen-tools.github.io/commitizen/), which comes pre-installed in the devcontainer.
+A pre-push commit hook will also run automatically to ensure that your commit messages follow this standard.
```

Comment:
```suggestion
A pre-push commit hook from [pre-commit](https://pre-commit.com/) may be used to automatically ensure your commit messages follow this standard.
```

Reply:
Removing the text about the pre-commit hook so we can discuss it further after 1.0 release.